scripts/netlify.js

```diff
@@ -0,0 +1,10 @@
+const shell = require('shelljs');
+
+shell.exec('git clone https://github.com/videojs/http-streaming');
```

Comment:
didn't see a better way to do this since vhs needs to build mux.js into itself.

Reply:
Git submodules! :trollface: